### ISSUE TITLE
Scaling service doesn't apply

### ISSUE THREAD

**Issue by jeremiahbowen** [aptible/terraform-provider-aptible#35]
We recently updated our process from using the cli in our pipelines to use the new terraform provider that y'all just released a few months ago. Kudos as the experience is awesome.
Here's a bug that we've encountered:
Have an app and service with 1 container count.
In terraform set the container_count to 0 and apply
The apps service container_count will not be updated in the environment.
Any subsequent terraform runs will show the update in the plan and not apply.
Update the container count manually (cli or ui).
Plan will no longer show a diff
Issue: Changes in terraform should apply without manual intervention
Terraform plan:
```
  # module.aptible-app["newrelic-prod"].aptible_app.app will be updated in-place
  ~ resource "aptible_app" "app" {
        id       = "20472"
        # (5 unchanged attributes hidden)
      + service {
          + container_count        = 0
          + container_memory_limit = 2048
          + process_type           = "cmd"
        }
      - service {
          - container_count        = 1 -> null
          - container_memory_limit = 2048 -> null
          - process_type           = "cmd" -> null
        }
    }
Plan: 0 to add, 1 to change, 0 to destroy.
```
CLI response after applying:
```
aptible services --environment pager-production  --app newrelic-prod
Id: 72423
Service: cmd
Command: CMD
Container Count: 1
Container Size: 2048
App: newrelic-prod
```

**Comment by jeremiahbowen** [comment]
Just another note that might be relevant. We're using the latest release of the provider: 0.1.2. Thanks for your help. Let me know if there's anything that we can do to help or test.

**Comment by robertfairhead** [comment]
I was able to recreate this and debug the issue today. The fix should be ready soon. When it is, I will release an updated version of this provider with it included.

The root cause is that the generated client for our API included a default `omitempty` annotation for the container count field. `0` is considered empty for integer values, therefore the updated value was not included in the API call, which caused it not to be updated.

Because the fix is in a dependency it will take a few days to get for that update to get reviewed and a new release there before I can include that updated version here to fix it.

I'm sorry for the delay in responding to this issue. I was on vacation for a few weeks at the end of year.

**Comment by robertfairhead** [comment]
The fix for this has been released in version v0.1.3, which now available on the Terraform Registry.
